.travis.yml

```diff
@@ -62,17 +62,17 @@ jobs:
             - libomp
           update: true
 
-    - name: macOS 10.15, GCC 10
+    - name: macOS 10.15, latest GCC (version 10)
       env:
-        - CC=/usr/local/opt/gcc@10/bin/gcc-10
-        - CXX=/usr/local/opt/gcc@10/bin/g++-10
+        - CC=/usr/local/opt/gcc/bin/gcc-10
+        - CXX=/usr/local/opt/gcc/bin/g++-10
```

Comment:
Is it possible to remove the `-10` suffixes from the compiler names? Otherwise this will break after the release of GCC 11 and I would instead suggest to stay at the fixed compiler version.

Reply:
No it is not unfortunately. We could create links of course - but this does not solve the problem. And we could switch back of course, but a break caused by a new release was the reason for this PR in the first place :(